packages/storybook/project.json

```diff
@@ -31,12 +31,12 @@
           },
           {
             "input": "packages/storybook",
-            "glob": "**/project-files-5/.storybook/**",
+            "glob": "**/project-files-ts/.storybook/**",
             "output": "/"
           },
           {
             "input": "packages/storybook",
-            "glob": "**/root-files-5/.storybook/**",
+            "glob": "**/root-files-ts/.storybook/**",
             "output": "/"
```

Comment:
Why is this listed in here twice? I don't think it should be.

Reply:
I don't think it is listed twice? 
<img width="416" alt="image" src="https://user-images.githubusercontent.com/12140467/173119928-8aa36803-810d-4b07-9e53-c8b557553198.png">

Reply:
🤦 Ignore me. I read root-files-ts and project-files-ts as the same folder.